src/pull_request.ts

```diff
@@ -278,3 +282,25 @@ async function submitReview(
     );
   }
 }
+
+function shouldIgnorePullRequest(pull_request: { body?: string }) {
+  const ignorePhrases = [
+    "@presubmit ignore",
+    "@presubmit: ignore",
+    "@presubmit skip",
+    "@presubmit: skip",
+    "@presubmitai ignore",
+    "@presubmitai: ignore",
+    "@presubmitai skip",
+    "@presubmitai: skip",
+  ];
```

Comment:
Instead of having a fixed list of phrases, consider using a regex pattern that would be more flexible and maintainable. This would also handle variations in whitespace more elegantly:

```typescript
const IGNORE_REGEX = /^@presubmit(?:ai)?(?:\s*:\s*|\s+)(ignore|skip)$/im;

function shouldIgnorePullRequest(pull_request: { body?: string }) {
  const body = pull_request.body ?? "";
  const matches = body.split('\n').some(line => IGNORE_REGEX.test(line.trim()));
  if (matches) {
    info('ignoring pull request due to ignore/skip command in description');
  }
  return matches;
}
```

This approach:
- Handles variations in whitespace more robustly
- Matches commands only at the start of a line (preventing false positives)
- Is more maintainable as it's a single pattern instead of multiple strings
- Can be easily extended to support additional variations